docs/guides/hooks-list.rst

```diff
@@ -304,6 +293,122 @@ Permission hooks
     Filters the result of ``elgg_entity_gatekeeper()`` to prevent access to an entity that user would otherwise have access to. A handler should return false to deny access to an entity.
 
 
+Subscriptions Notifications
+===========================
```

Comment:
We should say these are listed chronologically in the lifetime of the notification event, and similarly under the other heading.